pkg/image/image.go

```diff
@@ -49,8 +50,16 @@ func WithTags(tags ...string) AdditionalMetadata {
 		for _, t := range image.Metadata.Tags {
 			existingTags.Add(t.String())
 		}
+
 		for _, t := range tags {
-			tagObj, err := name.NewTag(t)
+			// it is possible that we are given references that have both a tag and a digest (or only one)
+			// we should only be allowing tags (stripping off digests if they are present)
+			fields := strings.Split(t, "@")
```

Comment:
would this be better to use `github.com/docker/distribution/reference` (or the other one) to parse the reference?

Reply:
I think I'll leave this one as is to get it over the finish line since the only thing it's doing is removing a digest if provided (and ignoring if there is no tag).